CriteoPublisherSdk/Sources/CR_HeaderBidding.m

```diff
@@ -213,7 +213,14 @@ - (void)addCriteoBidToDfpRequest:(id)adRequest
           customTargeting[CR_TargetingKey_crtSize] = [self stringSizeForBannerWithAdUnit:adUnit];
         }
 
-        NSString *dfpCompatibleString = [NSString cr_dfpCompatibleString:displayUrl];
+        NSString *dfpCompatibleString;
+        if (bid.isVideo) {
+          // No base64 encoding as there is no client javascript to decode
+          dfpCompatibleString = [[displayUrl cr_urlEncode] cr_urlEncode];
```

Comment:
Couldn't manage to do an end-to-end test with Irwan, but this double encode logic is directly coming from his tests.

Reply:
Ok we will need to check this once E2E test is possible. What is the blocker for such tests ?

Reply:
Basically ad manager requires a valid URL, which need a workaround / stub as we haven't implemented backend yet. Not a real E2E test w/o it. Will see while working on Test app integration how far this can be tested.